app/jobs/deposit_collection_job.rb

```diff
@@ -33,4 +36,19 @@ def perform_persist
       Sdr::Repository.register(cocina_object:)
     end
   end
+
+  def assign_participants(role)
```

Comment:
Can you add some code comments to this method?

Reply:
Also, where does this get tested? <-- This is my only critical question / comment.

Reply:
I was thinking this was tested in the create_collection system test, but I'll expand the job spec as well.